pandas/tests/frame/test_analytics.py

```diff
@@ -1274,3 +1274,28 @@ def test_series_broadcasting(self):
             df_nan.clip(lower=s, axis=0)
             for op in ["lt", "le", "gt", "ge", "eq", "ne"]:
                 getattr(df, op)(s_nan, axis=0)
+
+
+class TestDataFrameReductions:
```

Comment:
There is also a `tests/reductions/` directory with a lot of min/max tests. What is the general idea here of what kind of reductions tests should be placed where?

Reply:
tests.reduction is analogous to tests.arithmetic, intended for tests that are shared across classes

Reply:
hmm, you put the series ones in test_reductions.py I think these should be in a similar one?

Reply:
yah considered that, but for frame test_analytics contains most of the reduction tests.  i think the thing to do is get non-reduction tests out of there and then rename to test_reduction

Reply:
kk fair, keep in mind